src/app/components/code-block/index.html

```diff
@@ -0,0 +1,66 @@
+<sky-demo-page
+  pageTitle="Code block"
+  skyuxModule="SkyCodeBlockModule"
+  packageName="lib-code-block"
+>
+
+  <sky-demo-page-summary>
+    <p>
+      The code block component formats a block of text as a code block with syntax highlighting. The code block's header can display a copy button and language type. To create a code block, wrap <stache-code>sky-code-block</stache-code> tags around the text.
+    </p>
+    <p>
+      If a code block has text within <stache-code>&lt;</stache-code> and <stache-code>&gt;</stache-code> characters, then the component automatically appends a closing tag unless the code snippet is valid HTML. To prevent this, use "&#38;lt;" to escape <stache-code>&lt;</stache-code> and "&#38;gt;" to escape <stache-code>&gt;</stache-code>. For example, escape the <stache-code>&lt;</stache-code> and <stache-code>&gt;</stache-code> characters in the code snippet <stache-code>public Task&lt;Generic> MethodName()</stache-code> or the component automatically appends <stache-code>&lt;/Generic></stache-code> to the end: <stache-code>public Task&lt;Generic> MethodName()&lt;/Generic></stache-code>.
+    </p>
+    <p>
+      <sky-alert alertType="info">
+        To use the code block component, set it up as a dependency in your SKY UX SPA. First, install the package as part of your SPA dependencies: <stache-code>npm install @blackbaud/skyux-lib-code-block --save</stache-code>. Then install its plugin: <stache-code>npm install @blackbaud/skyux-builder-plugin-code-block --save</stache-code>. Next, add <stache-code>SkyCodeBlockModule</stache-code> as both an import and export in your <stache-code>app-extras.module.ts</stache-code> file. Finally, add the code block plugin to your <stache-code>skyuxconfig.json</stache-code> file: <stache-code>"plugins": ["@blackbaud/skyux-builder-plugin-code-block"]</stache-code>.
+      </sky-alert>
+    </p>
+  </sky-demo-page-summary>
+  
+  <sky-demo-page-properties>
+    <sky-demo-page-property
+      propertyName="code"
```

Comment:
Is this property to be used if you don't place code inside the component? Based on the demo I don't see how to use this property.

Reply:
This property isn't demonstrated in the Stache docs either. I'll follow up with Tim or Brandon on the specifics for this.

Reply:
Added the note @blackbaud-johnly mentioned in our Slack conversation